pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -2116,7 +2115,11 @@ func GracePeriodSeconds(vmi *v1.VirtualMachineInstance) int64 {
 	return gracePeriodSeconds
 }
 
-func InterpretTransitionalModelType(useVirtioTransitional *bool) string {
+func InterpretTransitionalModelType(useVirtioTransitional *bool, archString string) string {
+	// s390x uses ccw instead of pci
+	if isS390X(archString) {
+		return "virtio"
+	}
```

Comment:
This could perhaps benefit from a more detailed comment, for example something like:

> "virtio-non-transitional" and "virtio-transitional" are both PCI devices, so they don't work on s390x. "virtio" is a generic model that can be either PCI or CCW depending on the machine type

Reply:
I have added your suggestion for the comment.